### ISSUE TITLE
Improve event index / calendar page

### ISSUE THREAD

**Issue by ShaneCorry** [irishchessunion/icu_www_app#94]
Add field to select time controls
Add checkbox to only show fide-rated events
Add the map to the main page (_because the separate view is not so obvious_)

Improve screen reader layout (_there's some issues with nested label fields in the current form and the page lacks semantic elements_)
